activerecord/test/cases/core_test.rb

```diff
@@ -30,6 +30,11 @@ def test_inspect_limited_select_instance
     assert_equal %(#<Topic id: 1, title: "The First Topic">), Topic.all.merge!(select: "id, title", where: "id = 1").first.inspect
   end
 
+  def test_inspect_instance_with_non_primary_key_id_attribute
+    topic = topics(:first).becomes(TitlePrimaryKeyTopic)
+    assert_match /id: 1/, topic.inspect
```

Comment:
Can you address the warning?

```
% ARCONN=sqlite3 be ruby -w -Itest test/cases/core_test.rb
test/cases/core_test.rb:35: warning: ambiguous first argument; put parentheses or a space even after `/' operator
Using sqlite3
Run options: --seed 55120

# Running:

...........

Finished in 0.116315s, 94.5707 runs/s, 180.5441 assertions/s.
11 runs, 21 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
I fixed it. Thanks for catching that! I forgot that `bin/test` doesn't enable warnings by default 😅